doc/api/report.md

```diff
@@ -452,6 +456,9 @@ meaning of `SIGUSR2` for the said purposes.
 * `--report-signal` Sets or resets the signal for report generation
   (not supported on Windows). Default signal is `SIGUSR2`.
 
+* `--report-network-disabled` Disable the `header.networkInterfaces` reporting.
```

Comment:
I think "exclude" would be a more accurate name over "disabled", e.g. `--report-exclude-network`.
If we later on decide to allow more sections to be excluded from the report then we can follow a similar `--report-exclude-*` pattern, or have a single `--report-exclude=` option that takes a list of named sections to exclude.

Reply:
Love this idea. I'll rename it.